docker-app/qfieldcloud/subscription/models.py

```diff
@@ -524,10 +543,20 @@ class UpdateSubscriptionKwargs(TypedDict):
     @property
     @deprecated("Use `AbstractSubscription.active_storage_total_bytes` instead")
     def active_storage_total_mb(self) -> int:
+        if self.plan.is_seat_flexible:
+            return self.included_storage_mb + self.active_storage_package_mb
+
         return self.plan.storage_mb + self.active_storage_package_mb
 
     @property
     def active_storage_total_bytes(self) -> int:
+        if self.plan.is_seat_flexible:
+            return (
+                (self.included_storage_mb + self.active_storage_package_mb)
+                * 1000
+                * 1000
+            )
+
         return self.plan.storage_bytes + self.active_storage_package_bytes
```

Comment:
Can we move this logic within `Plans.included_storage_bytes`?

Reply:
You are right, I meant the `Subscription.included_storage_bytes`, there is no `Plan.included_storage_bytes`.